packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,646 @@
+        "required": [
+          "pageSize"
+        ],
+        "properties": {
+          "pageSize": {
+            "type": "number"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize",
+          "firstTxHash"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountAssetTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Asset Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "assetId",
+          "pageSize",
+          "firstTxHash"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetBlockRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Block.",
+        "type": "object",
+        "required": [
+          "height"
+        ],
+        "properties": {
+          "height": {
+            "type": "number"
+          }
+        }
+      },
+      "CallEngineRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Call Engine.",
+        "type": "object",
+        "required": [
+          "caller",
+          "callee",
+          "input"
+        ],
+        "properties": {
+          "type": {
+            "type": "number",
+            "enum": [0]
+          },
+          "caller": {
+            "type": "string"
+          },
+          "callee": {
+            "type": "string"
+          },
+          "input": {
+            "type": "string"
+          }
+        }
+      },
+      "GetEngineReceiptsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Engine Receipts.",
+        "type": "object",
+        "required": [
+          "txHash"
+        ],
+        "properties": {
+          "txHash":{
+            "type": "number"
+          }
+        }
+      },
+      "AddPeerRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Peer.",
+        "type": "object",
+        "required": [
+          "address",
+          "peerKey"
+        ],
```

Comment:
There is also optional TLS certificate: https://iroha.readthedocs.io/en/develop/develop/api/commands.html#add-peer